### ISSUE TITLE
Why choose us section UI Improvisation

### ISSUE THREAD

**Issue by riya-rjha** [itsmohit097/medi-hub#100]
The "Why Choose Us" section consists of hover effects over the boxes which are not looking good. The hover effect should be removed and the shadow should be permanent. Else the shadow should be added on hover but the rest of the boxes must not drift away from their positions. Either of the two would work & I'll show both the UI's. 

Also, there is another UI damage to it that after this section there is a little too lot space till the next upcoming section which can also be reduced to improve the UI.

Kindly assign this issue to me under GSSoC 2024 & I'll start working on it rightaway.

![Screenshot (570)](https://github.com/itsmohit097/medi-hub/assets/112813772/9ed6dec0-3f60-4779-a861-5498cc223330)

**Comment by itsmohit097** [comment]
@riya-rjha 
https://www.figma.com/design/eZiyFxl7zkLx9hE0Uz10zb/Untitled?node-id=1-14&t=Oq7OC61x2vartG1Y-1
i want to replace "wy choose us" with "our services"
are u up for this issue?

**Comment by riya-rjha** [comment]
Yes sure! I can work on that. Kindly assign me this issue under the Gssoc label. I'll start working rightaway.

**Comment by itsmohit097** [comment]
@riya-rjha assigned to u. Only make changes regarding to the issue (no other changes need to made)
change the  file name and make the appropriate changes.
[Important] : 
Do not waste your precious time. Kindly do the fork and branch process properly (read contribution.md file)
If not followed properly PR will not be merged and the issue will be assigned to another contributor.

**Comment by itsmohit097** [comment]
@riya-rjha update ??

**Comment by riya-rjha** [comment]
Yes I am working on it. Will update by today evening

**Comment by anshikatiwari20** [comment]
**Description:**
The enhancement involves adding an "Our Services" section to the homepage. This section will provide an overview of the services offered by the project, enhancing user understanding and engagement.

I am a GSSoC contributor and would like to work on adding the "Our Services" section to the homepage as described in issue #100. Could you please assign this issue to me? Thank you!

**Comment by anshikatiwari20** [comment]
Description:
The enhancement involves adding an "Our Services" section to the homepage. This section will provide an overview of the services offered by the project, enhancing user understanding and engagement.

@itsmohit097 I am a GSSoC contributor and would like to work on adding the "Our Services" section to the homepage as described in issue https://github.com/itsmohit097/medi-hub/issues/100. Could you please assign this issue to me? Thank you!
